### PR TITLE
Let example/demo apps run as standalone apps.

### PR DESCRIPTION
**Pull request by mkcor**
Lays the ground for #133.

With this PR, it is still possible to run `python index.py` from the root of the repo, but now also possible to run `python tests/dashbio_demos/app_ideogram.py`.

What's left to do:

- [x]  I have run `pip install -i https://test.pypi.org/simple/ dash-bio` in my (activated) virtualenv, but I'm not sure about the syntax to add it to `tests/requirements.txt`...

- [x]  Read again something like https://chrisyeh96.github.io/2017/08/08/definitive-guide-python-imports.html because imports are of different kinds in these changesets and I'm confused...

- [ ]  Add the same `if __name__ == '__main__':` block to all other example apps.

### CODE REVIEW

**Comment by mkcor** [comment]
So, the fail on CircleCI is due to a pylint error already documented by @Bachibouzouk in this comment: https://github.com/plotly/dash-bio/pull/144#pullrequestreview-198624734

If I add the `.` in front the relative(?) import, I cannot run `python tests/dashbio_demos/app_ideogram.py` anymore:
```bash
Traceback (most recent call last):
  File "tests/dashbio_demos/app_ideogram.py", line 860, in <module>
    from .utils.app_standalone import run_standalone_app
ModuleNotFoundError: No module named '__main__.utils'; '__main__' is not a package
```
which is why I need to read again the Definitive Guide to Python Imports (or get some sleep).

**Comment by shammamah** [comment]
@mkcor You just need to add 
```
--extra-index-url https://testpypi.pypi.org/simple/
```
to the beginning of `requirements.txt`. (Reference: https://stackoverflow.com/questions/29289695/installing-packages-from-multiple-servers-from-one-or-more-requirements-file)

**Comment by mkcor** [comment]
> @mkcor You just need to add
> 
> ```
> --extra-index-url https://testpypi.pypi.org/simple/
> ```

@shammamah Oh, just like you did at the root-level `requirements.txt`! I have taken this opportunity to fix up the `dash-bio` dependency (not to be confused with the `dash_bio` import, as we were discussing!) and bump up its version. Is that okay?

**Comment by mkcor** [comment]
After studying relative/absolute module/package imports for the n-th time, I'm reminded that there is more than one way to do them... So I went for tweaking `.pylintrc` instead! I cherry-picked the corresponding commit from the previous, work-in-progress branch.

File: requirements.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ biopython
 colour==0.1.5
 cython>=0.19
 dash==0.35.2
-dash_bio==0.0.1
+dash-bio==0.0.2
```
Comment (shammamah) [inline requirements.txt]:
```suggestion
dash-bio==0.0.3
```

File: tests/requirements.txt [hunk 2]
```diff
@@ -6,6 +8,7 @@ chromedriver-binary==2.45.0
 colour==0.1.5
 cython>=0.19
 dash==0.35.2
+dash-bio==0.0.2
```
Comment (shammamah) [inline tests/requirements.txt]:
```suggestion
dash-bio==0.0.3
```

File: tests/dashbio_demos/utils/app_wrapper.py [hunk 3]
```diff
@@ -5,8 +5,9 @@
 
 def app_page_layout(page_layout,
                     app_title="Dash Bio App",
-                    app_github_url="",
```
Comment (Bachibouzouk) [inline tests/dashbio_demos/utils/app_wrapper.py]: @mkcor  if this argument is renamed here, it should be refactored in `index.py` as well, this is what make the Integration tests fail on CircleCI

**Comment by mkcor** [comment]
@Bachibouzouk did you see my comment https://github.com/plotly/dash-bio/pull/149#issuecomment-460465735 and the related commit f0b6f5d? This comment updated the earlier one about CircleCI tests failing (where I tagged you), and this commit made the CI tests pass. The CI tests were passing until I bumped the version number for dash-bio (commit 8105bc9).

**Comment by mkcor** [comment]
@shammamah Please write short enough commit messages (and without typos).

File: tests/dashbio_demos/utils/app_wrapper.py (hunk 3 above)
Comment (shammamah) [reply tests/dashbio_demos/utils/app_wrapper.py]: Fixed in https://github.com/plotly/dash-bio/pull/149/commits/531a8885586fa794eb97be6394aa9f8a626d09e4.

**Comment by mkcor** [comment]
Thank you so much, @shammamah! Can you approve (even if you authored the latest commits...)? I'll merge since I originally submitted the PR.

**Pull request merged**